spring-cloud-dataflow-server/docker-compose-cf.yml

```diff
@@ -0,0 +1,17 @@
+version: '3'
+
+services:
+
+  skipper-server:
```

Comment:
I'm not there's much value in keeping these standalone files. IMO, no one would be in a position to start SCDF and everything in locally running Docker daemon, but instead deploy streams/tasks to a cloud platform. I know we wanted to showcase the possibility, but I wonder if it would end up confusing the users.

Reply:
I believe that it is useful especially in combination with `DockerComposeIT` like testing. It will be awkward to explain how to do this on the DockerComposeIT javadoc. 
Instead i've added it as a  bullet to the updated customized docker-compose guide (still not pushed)

Reply:
I mean after i push the changes to the microsite will update the java doc to reference multiplatform support.

Reply:
I've pushed a commit with simplified miltiplatform testing explanation, referring to https://dataflow.spring.io/docs/2.3.0.SNAPSHOT/installation/local/docker-customize/